_pages-partials/getting-started/05-controls.html.twig

```diff
@@ -0,0 +1,226 @@
+    </p>
+
+    <article class="mb-4">
+        <h3 id="tank-controls" class="border-b pb-1 mb-3">Tank Controls</h3>
+
+        <p>Basic controls for moving your tank and playing.</p>
+
+        <div class="c-well my-3">
+            <div class="row text-center">
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Movement</p>
+                        <div class="controls">
+                            <div class="control">
+                                {% include '_includes/keyboard_arrows.html.twig' %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Spawn</p>
+
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key">I</kbd>
+                            </div>
+                            <div class="control">
+                                {% include '_includes/mouse.html.twig' with { pressed: 'right' } %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Shoot</p>
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key enter" data-align="right">
+                                    enter
+                                </kbd>
+                            </div>
+                            <div class="control">
+                                {% include '_includes/mouse.html.twig' with { pressed: 'left' } %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Jump</p>
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key tab" data-align="left">
+                                    tab
+                                </kbd>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">
+                            Identify / Lock On
+                            <abbr title="with Guided Missile flag">*</abbr>
```

Comment:
I'm abusing `<abbr>` here, let's make this a `<span>` with a `title` and a class to style it like an `<abbr>`.